InvoiceMaker/InvoiceMaker/Views/Clients/Create.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@model InvoiceMaker.FormModels.CreateClient
+
+<h1>Create a Client</h1>
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to the POST action method.